v20.1/backup-and-restore.md

```diff
@@ -1,67 +1,96 @@
 ---
 title: Back up and Restore Data
-summary: Learn how to back up and restore a CockroachDB database.
+summary: Learn how to back up and restore a CockroachDB cluster.
 toc: true
 redirect_from:
 - back-up-data.html
 - restore-data.html
 ---
 
-Because CockroachDB is designed with high fault tolerance, backups are primarily needed for disaster recovery (i.e., if your cluster loses a majority of its nodes). Isolated issues (such as small-scale node outages) do not require any intervention. However, as an operational best practice, we recommend taking regular backups of your data.
+Because CockroachDB is designed with high fault tolerance, backups are primarily needed for disaster recovery (i.e., if your cluster loses a majority of its nodes). Isolated issues (such as small-scale node outages) do not require any intervention. However, as an operational best practice, **we recommend taking regular backups of your data**.
 
-Based on your [license type](https://www.cockroachlabs.com/pricing/), CockroachDB offers two methods to back up and restore your cluster's data: Enterprise and Core.
+Based on your [license type](https://www.cockroachlabs.com/pricing/), CockroachDB offers two methods to backup and restore your cluster's data: [Enterprise](#perform-enterprise-backup-and-restore) and [Core](#perform-core-backup-and-restore).
 
 ## Perform Enterprise backup and restore
 
 If you have an [Enterprise license](enterprise-licensing.html), you can use the [`BACKUP`][backup] statement to efficiently back up your cluster's schemas and data to popular cloud services such as AWS S3, Google Cloud Storage, or NFS, and the [`RESTORE`][restore] statement to efficiently restore schema and data as necessary.
 
-### Manual full backups
+{{site.data.alerts.callout_success}}
+We recommend [automating daily backups of your cluster](#automated-full-and-incremental-backups). To automate backups, you must have a client send the `BACKUP` statement to the cluster. Once the backup is complete, your client will receive a `BACKUP` response.
+{{site.data.alerts.end}}
+
+### Full backups
+
+In most cases, **it's recommended to take full nightly backups of your cluster**. A full cluster backup allows you to do the following:
+
+- Restore a full cluster
+- Restore database(s) from the cluster
+- Restore table(s) from the cluster
+
+To do a full cluster backup, use the [`BACKUP`](backup.html) statement:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> BACKUP TO '<backup_location>';
+~~~
+
+If it's ever necessary, you can use the [`RESTORE`][restore] statement to restore your full cluster:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> RESTORE FROM '<backup_location>';
+~~~
 
-In most cases, it's recommended to use the [`BACKUP`][backup] command to take full nightly backups of your cluster:
+{{site.data.alerts.callout_info}}
+Note that a full cluster restore can only be run on a target cluster with no databases or tables.
```

Comment:
Technically it's "user created databases or tables" (since with a new cluster, you have system tables, defaultdb, etc.)

Reply:
Fixed!